apps/accounts/api/automations/change_secret_dashboard.py

```diff
@@ -159,8 +155,7 @@ def get(self, request, *args, **kwargs):
                         if name == self.task_name and tp == self.tp:
                             execution_ids.append(_id)
 
-            snapshots = self.change_secret_executions_queryset.filter(
-                id__in=execution_ids).values_list('id', 'snapshot')
+            snapshots = AutomationExecution.objects.filter(id__in=execution_ids).values_list('id', 'snapshot')
 
             asset_ids = {asset for i in snapshots for asset in i.get('assets', [])}
             account_ids = {account for i in snapshots for account in i.get('accounts', [])}
```

Comment:
There are no changes that need to be addressed at this time. Everything looks fine with the provided code. If you have any additional questions, please feel free to ask!